src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -395,8 +395,12 @@ else if (item == PrisonEscapeItem.SEARCH) {
 	
 	private void arrestPlayer(PrisonEscapePlayer arrested, PrisonEscapePlayer arrester) {
 		teleportToSolitary(arrested);
-		
-		// TODO warn players
+
+		for (PrisonEscapePlayer player : _playersOnLobby) {
+			MessageLanguageManager messages = MessageLanguageManager.getInstance(player.getName());
```

Comment:
This method receives as argument the language. Use getInstanceByPlayer method instead. (I am not sure if the method name is exactly that).